oom-frontend/src/app/shared/modules/collaborator/collaborator-editor/collaborator-editor.component.ts

```diff
@@ -0,0 +1,38 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {User} from "../../../models/user.model";
+import {UsersBackendService} from "../../../../core/users/users-backend.service";
+
+@Component({
+  selector: 'collaborator-editor',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
  selector: "collaborator-editor",
```